pycolmap/pycolmap/geometry/homography_matrix.h

```diff
@@ -0,0 +1,41 @@
+#include "colmap/geometry/homography_matrix.h"

```

Comment:
Do we need a pragma once here and elsewhere?

Reply:
Most files are meant to be included only once so I've added pragma once only on helpers that are included multiple times - is that ok?

Reply:
I would still add the pragma once. It doesn't hurt?

Reply:
Done.